README.md

```diff
@@ -44,6 +44,8 @@ conda create -y -n gym-kmanip python=3.10 && conda activate gym-kmanip
 pip install -e .
 ```
 
+There are some asset files that are not yet in source control. Download the zip file `https://drive.google.com/file/d/1PNsBaPHwvQuqndZD64N8q6U9eNiU-uiM/view?usp=sharing` and extract the contents to 'gym_kmanip/assets' folder relative to the root of repository (which should also be a folder named gym_kmanip)
```

Comment:
We don't need this - we will push the latest stompy this weekend.

Reply:
Although we could explicitly give a link to the s3 bucket with latest stompy arm

Reply:
Can you tell me the s3 bucket? I can put the reference to s3_bucket rather than the drive location.

Anyone starting with the repository need to know and do this step for tests to pass. Hence that needs to be there (unless the assets are added to git OR downloaded to repository through some other step).

Of course, this ReadMe has become stale with later steps too. Work for later.

Reply:
Good point, coming!